python/cuml/linear_model/logistic_regression.pyx

```diff
@@ -254,9 +254,9 @@ class LogisticRegression(Base):
         y_m, _, _, _, _ = input_to_dev_array(y)
 
         unique_labels = rmm_cupy_ary(cp.unique, y_m)
-        num_classes = len(unique_labels)
+        self.num_classes = len(unique_labels)
```

Comment:
Since this is an internal variable, I would suffix it with `_` to match other sklearn conventions.